src/app/pages/storage/volumes/volume-list-controls/volumes-list-controls.component.scss

```diff
@@ -6,3 +6,10 @@
 .spinner {
   margin-left: 8px;
 }
+
+.ix-form-container {
+  ix-input {
```

Comment:
Can we improve styles to have the same height for all elements on the toolbar? 
![image](https://user-images.githubusercontent.com/351613/142658261-74b491cf-9fa2-4a04-8275-98f54dc5a267.png)
![image](https://user-images.githubusercontent.com/351613/142658283-71e62d32-4970-4e52-86a0-a0f05e8f50e0.png)

Entity Table Toolbar, for example:
![image](https://user-images.githubusercontent.com/351613/142660458-a91faac0-8563-40c7-8242-8063661c6d32.png)